client/src/scripts/managers/inputManager.ts

```diff
@@ -342,6 +343,50 @@ export class InputManager {
                 shootOnRelease = false;
             });
         }
+        const ticker = new PIXI.Ticker();
+        ticker.stop();
+        ticker.add(() => {
+            const gamepads = navigator.getGamepads();
+            if (gamepads[0]) {
+                const leftJoystickMoving = gamepads[0].axes[0] !== 0 || gamepads[0].axes[1] !== 0;
+                const rightJoystickMoving = gamepads[0].axes[2] !== 0 || gamepads[0].axes[3] !== 0;
+                // const rightJoystickDistance = Math.sqrt(gamepads[0].axes[2] * gamepads[0].axes[2] + gamepads[0].axes[3] * gamepads[0].axes[3]);
+                // distance formula for stuff like throwables, USAS-12, and M590M
+                if (leftJoystickMoving) {
```

Comment:
probably best to allow players to configure which joystick to use for what

Reply:
Added